crates/litesvm/src/lib.rs

```diff
@@ -465,6 +465,7 @@ impl LiteSVM {
             .collect::<Vec<&u8>>();
         let fee = solana_fee::calculate_fee(
             message,
+            false,
```

Comment:
@kevinheavey I'm afraid of this change. Maybe it will break something on the previous 2.1.*

Reply:
Is this change here because solana-fee changed its API between 2.1.0 and 2.1.7?

Reply:
It's appear in 2.1.1 version of the solana-fee crate